From fab7e959ff798466b493ff90e2248078ae37a2dc Mon Sep 17 00:00:00 2001
From: Francisco Franco <franciscofranco.1990@gmail.com>
Date: Sat, 15 Jun 2013 21:11:06 +0100
Subject: [PATCH 008/517] writeback: I believe these defines were calculated
 with HZ=1000 in mind. Since our devices are HZ=100
 I hardcoded the values on this constants. I'm not
 100% sure if this code was made for HZ with 1000
 but I couldn't track it to understand clearly.
 Although I didn't see any drawback of using this as
 it is now. If anyone knows the answer please chime
 in.

Signed-off-by: Francisco Franco <franciscofranco.1990@gmail.com>
Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 mm/page-writeback.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mm/page-writeback.c b/mm/page-writeback.c
index 07701c5..aae8b87 100644
--- a/mm/page-writeback.c
+++ b/mm/page-writeback.c
@@ -39,7 +39,7 @@
 /*
  * Sleep at most 200ms at a time in balance_dirty_pages().
  */
-#define MAX_PAUSE		max(HZ/5, 1)
+#define MAX_PAUSE		200
 
 /*
  * Try to keep balance_dirty_pages() call intervals higher than this many pages
@@ -50,7 +50,7 @@
 /*
  * Estimate write bandwidth at 200ms intervals.
  */
-#define BANDWIDTH_INTERVAL	max(HZ/5, 1)
+#define BANDWIDTH_INTERVAL	200
 
 #define RATELIMIT_CALC_SHIFT	10
 
-- 
1.7.9.5

